testsrc/test262.properties

```diff
@@ -1432,19 +1440,16 @@ built-ins/RegExp 925/1464 (63.18%)
     prototype/source/value-empty.js
     prototype/source/value-line-terminator.js
     prototype/source/value-u.js
-    prototype/sticky 8/8 (100.0%)
+    prototype/sticky/cross-realm.js {unsupported: [cross-realm]}
+    prototype/sticky/length.js
```

Comment:
BTW: this test + the next 2 fail because the 'sticky' property you added is of the wrong type.

Having said that, so are all the existing properties like multiline for example.

Not sure if we should continue as is, just do this new property properly (and fix the other ones later) or fix them all now

Reply:
@p-bakker what do you mean by wrong type?

Reply:
See https://github.com/mozilla/rhino/issues/963: the global/multiline/ignoreCase properties and now the new sticky property ought to be accessor properties, but they are implemented as data properties

Reply:
Got no problem with that.

Reply:
Once you merge I'll rebase my Regex unicode flag WIP and create a draft PR for it (not 100% complete yet)

Reply:
Merged, fix for the proprties is also on the way